docs/MD.md

```diff
@@ -1,29 +1,129 @@
+* Some entries do not have a part of speech indicated.
+* There are 47 unique parts of speech in the database. This list will need to be reduced and standardized.
+
+Abbreviation                     | Part of Speech
+---------------------------------|---------------
+anim pl pron                     | animate plural pronoun
+anim/prefix                      |
+bp                               |
+excl                             | exclamation
+exclamation                      | exclamation
+expr                             |
+fv                               |
+fvp                              |
+gvp                              |
+inan pl pron                     | inanimate plural pronoun
+interj                           | interjection
+interjection                     | interjection
+IPC                              | indeclinable particle
+loc                              |
+n                                | noun
+n pl                             | plural-only noun
+n sg                             | singular-only noun
+particle denoting the past tense | particle denoting past tense
+phrase                           | phrase
+prefix                           | prefix
+prefix denoting something happen | prefix denoting something happened
+preverb                          | preverb
+pron                             | pronoun
+pron pl                          | plural pronoun
+pron plural                      | plural pronoun
+pron sing                        | singular pronoun
+ques                             | question
+question                         | question
+reduplicative prefix             | reduplicative prefix
+reply                            | reply
+suffix                           | suffix
+v                                | verb
+v comb                           |
+v command                        | imperative verb
+v phrase                         |
+v phrase pl                      |
+v pl                             | plural-only verb
+v sg                             | singular-only verb
+vc                               |
+vc pl                            |
+vcp                              |
+vfp                              |
+vgp                              |
+vp                               |
+vp pl.                           |
+vpq                              |
+
+## MeaningInEnglish
 
-## Definition
+The English definition of the term.
 
+* Occasionally includes example phrases or sentences, usually preceded by `e.g.` / `E.g.`.
+  - Other types of notes are also preceded by `e.g.` / `E.g.`, making it difficult to rely on this for programmatic processing.
+  - Example phrases are followed by their translations. If the example ends in punctuation, the translation simply follows the example, after a space. If the example does not end in punctuation, the example and its translation are separated by a hyphen surrounded by spaces.
+  - **ex:** "You are cold. Usually used in a question. E.g. kikawacin ci? Are you cold?"
```

Comment:
The orthography of these example words/sentences are not according to SRO (in particular vowel length).

Reply:
Right—we'll need to identify stretches of Cree within definitions and standardize them.